.github/workflows/build_and_test.yml

```diff
@@ -47,13 +47,31 @@ on:
         required: false
         type: boolean
         default: true
+      IS_OPTIONAL:
+        required: false
+        type: boolean
+        default: false
+      REQUIRED_LABEL:
+        required: false
+        type: string
     secrets:
       GOOGLE_CLOUD_GCP:
         required: false
 jobs:
   build_test_deploy:
     runs-on: ${{ inputs.RUNS_ON }}
     steps:
+    - name: Check labels
```

Comment:
Can you update the label such that its intent is clearer for a new comer?

Reply:
Can I go ahead and change the labels in Guthub? I think Randy and Matteo were onboard with this as well. So changing the `ci: ready to be merged` to something indicating it runs the CUDA builds?

Reply:
As you wish, it's this issue https://github.com/GEOS-DEV/GEOS/issues/3000

Reply:
But be careful to be in sync. When you merge you'll need to rename the `label`.
Plus I was also referring to the `- name: Check labels` which is not so obvious.

Reply:
I have made the name of the step more descriptive. Perhaps its easier to change the label name in a separate PR just for clarity.